scripts/rnnlm/initialize_matrix.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+# Copyright  2017  Jian Wang
+#            2017  Daniel Povey
+# License: Apache 2.0.
+
+import argparse
+import sys
+import random
+import math
+
+parser = argparse.ArgumentParser(description="This script is used to randomly initialize "
+                                 "Kaldi-format matrices, with supplied dimensions and "
+                                 "standard deviation.",
+                                 epilog="E.g. " + sys.argv[0] + " --num-rows=10 --num-cols=20 "
+                                 "--param-stddev=0.05 > exp/rnnlm1/embedding.mat",
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+parser.add_argument("--num-rows", type=int, required=True,
```

Comment:
BTW I did a quick test and it looks like perl is only a factor of 3 faster than python for this type of thing, but I think that's enough for this purpose.